.github/workflows/dispatch_deploy.yml

```diff
@@ -38,7 +38,7 @@ permissions:
 jobs:
   get_deployed_version:
     name: Lookup Version
-    if: ${{ !inputs.deploy-app && inputs.version == '-current-' }}
+    if: ${{ !inputs.deploy-app && inputs.version == '-latest-' }}
```

Comment:
Shouldn't this still be `-current-`?

Reply:
That makes sense.
I've made changes to the workflow to use `latest`, `current`, and manual tag when choosing the version. So we can use all of them as an option now.